### ISSUE TITLE
Define a Measure object with derivative???

### ISSUE THREAD

**Issue by giuseppec** [mlr-org/measures#2]
mboost, xgboost etc. sometimes require a loss function + its gradient. If we define a Measures object we probably can add an (optional) slot that contains the gradient so that other packages such as mboost, xgboost can use it???

**Comment by PhilippPro** [comment]
But this has to be a function, at least for xgboost, no? Maybe just define a new function (e.g. ACCGradient) which returns a list consisting of gradient and second order gradient? for the specific measure, and then it can be used for xgboost?
